inference-engine/tests/functional/inference_engine/transformations/convert_reduce_to_pooling_test.cpp

```diff
@@ -102,9 +102,15 @@ class ConvertReduceToPoolingTests: public CommonTestUtils::TestsCommon,
 
         // TODO: handle multiply_value for ReduceSum case when set_keep_dims is ready
 
-        if (!params.reshape_end.empty()) {
+        //Convert std::vector<int64_t> -> ov::Shape
+        ov::Shape _reshape_end;
+        for (auto v : params.reshape_end) {
+            _reshape_end.push_back(v);
+        }
```

Comment:
```suggestion
        ov::Shape reshape_end(params.reshape_end.begin(), params.reshape_end.end());
```
BTW what is the purpose of underscore `_` in variable name?

Reply:
fixed